skynet/modules/ttt/summaries/prompts/summary.py

```diff
@@ -1,6 +1 @@
-summary_template = """
-<s>[INST] <<SYS>>
-Provide a short summary of the following text.
-<</SYS>>
-{text} [/INST]
-"""
+summary_system_message = "Provide a concise summary of the following text."
```

Comment:
Note for the future: let's add a "hint" parameter to the summaries API and have it be "conversation" or "text" and have different prompts.

Reply:
I didn't want to change this significantly without testing it heavily first, since this is what's primarily being used out there

Reply:
We could make "text" the default and keep this prompt. Just an idea.